libraries/rib-base/src/test/java/com/badoo/ribs/clienthelper/childaware/ChildAwareImplTest.kt

```diff
@@ -198,7 +198,8 @@ class ChildAwareImplTest {
 
     @Test
     fun `ignores registration when parent lifecycle is destroyed`() {
-        val testLifecycle = TestLifecycle(Lifecycle.State.DESTROYED)
+        val testLifecycle = TestLifecycle(Lifecycle.State.CREATED)
```

Comment:
This is the main change in quite a few of the tests. You cannot go for INITIALISED to DESTROYED. Perhaps some of these tests are no longer valid and should be removed. Happy to discuss!